tests/src/CompilerTest.php

```diff
@@ -767,4 +794,35 @@ private function verify($scenarioJson, $method, $arguments, $mysql, $phpInput, $
             );
         }
     }
+
+    private function verifyException($scenarioJson, $method, $arguments, $code, $data)
+    {
+        // cinnabari stuff
```

Comment:
I believe that the best comments explain the "why" behind the code.

Anyone can see _what_ the code does: a "what" comment would just break the flow and clutter things up.

But, sometimes, even when it's clear what the code does, it's not clear _why_ it was written that way. In that case (assuming the code is well written), then we need a "why" comment... and people will be genuinely grateful that you wrote it!

Is this a "what" comment or a "why" comment?